### PR TITLE
Neue 6.0 beta4

### PR DESCRIPTION
**Pull request by DFurnes**
# Changes

Another messy one. This should be the last.
- Overrides theme functions to properly render `.field-label`, `.text-field`, `.text-field.-search`, `.select`, `.option` patterns from Form API.
- Uses new `list` pattern (and `.with-lists` wrapper for Markdown content) to apply list styles where necessary, and `-compacted` and `-concatenated` modifiers where necessary. This replaces the old `list-compacted` override we were using.
- Uses new `form-actions` pattern introduced in this release.
- Uses renamed `container__block` instead of `container__body`.

Ready for review! :cactus: 
For review: @DoSomething/front-end

### CODE REVIEW

File: lib/themes/dosomething/paraneue_dosomething/includes/form.inc [hunk 1]
```diff
@@ -13,6 +13,133 @@ function paraneue_dosomething_form_alter_base(&$form, &$form_state, $form_id) {
...
 
 /**
+ * Implements theme_form_element().
+ */
+function paraneue_dosomething_form_element($variables) {
+  $element = &$variables['element'];
+  $output = theme_form_element($variables);
+
+  // If rendering a radio button, create rendered element for label
+  if($element['#type'] == 'radio') {
```
Comment (weerd) [inline lib/themes/dosomething/paraneue_dosomething/includes/form.inc]: Better to use `===`?

File: lib/themes/dosomething/paraneue_dosomething/includes/form.inc [hunk 2]
```diff
@@ -13,6 +13,133 @@ function paraneue_dosomething_form_alter_base(&$form, &$form_state, $form_id) {
...
+    $variables['rendered_element'] = $element['#children'] . '<span class="option__indicator"></span>' . "\n";
+    $output = theme('form_element_label', $variables);
+  }
+
+  // If rendering a checkbox, create rendered element for label
+  if($element['#type'] == 'checkbox') {
```
Comment (weerd) [inline lib/themes/dosomething/paraneue_dosomething/includes/form.inc]: Same as above.

File: lib/themes/dosomething/paraneue_dosomething/includes/form.inc (hunk 1 above)
Comment (DFurnes) [inline lib/themes/dosomething/paraneue_dosomething/includes/form.inc]: In PHP too? Seems like we generally use `==`.

File: lib/themes/dosomething/paraneue_dosomething/templates/system/html.tpl.php [hunk 3]
```diff
@@ -19,7 +19,7 @@
   <meta name="crowdcurity-site-verification" content="voJ0lyI8JXV5Ab1XAYMaoQj7WsoScgkfk9TLG0dA">
 
   <title><?php print $head_title; ?></title>
-  <script type="text/javascript" src="<?php print NEUE_ASSET_PATH; ?>/dist/modernizr-neue.js"></script>
+  <script type="text/javascript" src="<?php print DS_ASSET_PATH; ?>/dist/modernizr.js"></script>
```
Comment (weerd) [inline lib/themes/dosomething/paraneue_dosomething/templates/system/html.tpl.php:22]: :dancer:

File: lib/themes/dosomething/paraneue_dosomething/includes/form.inc (hunk 1 above)
Comment (weerd) [inline lib/themes/dosomething/paraneue_dosomething/includes/form.inc]: PHP also uses `===` to check type.

Comment (aaronschachter) [inline lib/themes/dosomething/paraneue_dosomething/includes/form.inc]: @sergii-tkachenko has been reviewing PR's pushing for the `===` too

**Comment by weerd** [comment]
Overall looks grand! :100: :sheep: IT!

File: lib/themes/dosomething/paraneue_dosomething/includes/form.inc (hunk 1 above)
Comment (DFurnes) [inline lib/themes/dosomething/paraneue_dosomething/includes/form.inc]: :+1: Will change!

**Pull request merged**